src/widgets/LangSwitcher.php

```diff
@@ -184,10 +184,15 @@ public function run()
 
         $items = [];
 
+        if (!Yii::$app->composition->hidden && property_exists(Yii::$app->composition, 'hideDefaultOnly')) {
+            $hideDefaultOnly = Yii::$app->composition->hideDefaultOnly;
+            Yii::$app->composition->hideDefaultOnly = false;
```

Comment:
Why do have to set the property to false at this point?

Reply:
if it's not off then widget will not add prefixes to links in switch widget.

Reply:
default language would still have correct, non-prefixed, link.